backend/src/services/auth/auth.service.ts

```diff
@@ -29,4 +30,26 @@ const signupLocal = async (
   };
 };
 
-export { signupLocal };
+const signinLocal = async (user: User): Promise<SignInResponseData> => {
+  const { email } = user;
+
+  const accessTokenPayload = {
+    email,
+    sub: 'accessToken',
+  };
+
+  const refreshTokenPayload = {
+    email,
+    sub: 'refreshToken',
+  };
```

Comment:
There's a convention that we put the `user.id` as the `sub` in the JWT tokens. Because `sub` should be a unique identifier for the entity. And the entity here is the user
![image](https://user-images.githubusercontent.com/68850090/183850253-472f1f53-b1ce-484d-9f57-0402f88b342c.png)

Reply:
done